tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -748,9 +748,18 @@ void SpirvEmitter::HandleTranslationUnit(ASTContext &context) {
     // TODO: assign specific StageVars w.r.t. to entry point
     const FunctionInfo *entryInfo = workQueue[i];
     assert(entryInfo->isEntryFunction);
+
```

Comment:
Oh .. this method becomes long. Could you please separate this part?
e.g.,
```
llvm::StringRef SpirvEmitter::getEntryPointName() {
    llvm::StringRef entrypointName = entryInfo->funcDecl->getName();
    // If this is the -E HLSL entrypoint and -fspv-entrypoint-name was set,
    // rename the SPIR-V entrypoint.
    if (entrypointName ==
            theCompilerInstance.getCodeGenOpts().HLSLEntryFunction &&
        !spirvOptions.entrypointName.empty()) {
      entrypointName = spirvOptions.entrypointName;
    }
  return entrypointName;
}
```

Reply:
(sorry for making it long before .. let's make it shorter)

Reply:
Done. Also noticed there was already a member variable for `HLSLEntryFunction` so I cleaned that usage up.